src/components/common/LikeButton.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import likeImg from "../../assets/like.png";
+
+export default function LikeButton({
+  like,
+  updateVotes,
+  patchTarget,
+  targetId,
+}) {
+  const [selected, setSelected] = useState(false);
+  const [isError, setIsError] = useState(false);
```

Comment:
To keep the naming consistent, selected should be called isSelected 
(this helps readability and helps you figure out the data type)